freqtrade/optimize/hyperopt_interface.py

```diff
@@ -36,6 +36,12 @@ class IHyperOpt(ABC):
     """
     ticker_interval: str
 
+    def __init__(self, config: dict) -> None:
+        self.config = config
+
+        # Assign ticker_interval to be used in hyperopt
+        IHyperOpt.ticker_interval = str(config['ticker_interval'])
```

Comment:
Consider i've implemented my own constructor in the child-hyperopt-class and don't call `super().__init__(config)` (it's not documented anywhere since we don't use this by default, so how would i know?) so config and ticker_interval are not set (although i would expect them to be since i use them ... )

It can become a silent, odd failure that'll be very difficult to debug.

Now on the other hand, we do the same with the Strategy Interface and have had no problems related to it so far ...